section2/exercises/if-statements.js

```diff
@@ -82,14 +92,19 @@ When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pi
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
 
-var cupsOfFlour = 1;
+var cupsOfFlour = 2;
 var hasSauce = true;
 
 // Write your conditional here
-
+if (cupsOfFlour >= 2 && hasSauce == true){
```

Comment:
Great solution!

You will want to bring that `else` up on the the same line as the closing curly brace right before it. Same for line 77/78